ui/packages/consul-ui/app/models/node.js

```diff
@@ -1,27 +1,27 @@
-import Model from 'ember-data/model';
-import attr from 'ember-data/attr';
+import Model, { attr } from '@ember-data/model';
 import { computed } from '@ember/object';
 
 export const PRIMARY_KEY = 'uid';
 export const SLUG_KEY = 'ID';
 
-export default Model.extend({
-  [PRIMARY_KEY]: attr('string'),
-  [SLUG_KEY]: attr('string'),
-  Address: attr('string'),
-  Node: attr('string'),
-  Meta: attr(),
-  Services: attr(),
-  Checks: attr(),
-  CreateIndex: attr('number'),
-  ModifyIndex: attr('number'),
-  TaggedAddresses: attr(),
-  Datacenter: attr('string'),
-  Segment: attr(),
-  Coord: attr(),
```

Comment:
I took out Coord and Segment from here, they seem to have been mistakenly added here from a long long time ago.